os/board/rtl8720e/src/component/bluetooth/os/tizenrt/osif_tizenrt.c

```diff
@@ -202,6 +202,11 @@ bool osif_task_priority_get(void *p_handle, uint16_t *p_priority)
 
 	p_tcb = sched_gettcb(pid);
 
+	if (p_tcb) {

```

Comment:
Should this be if(p_tcb == NULL) or if(!p_tcb)?

Reply:
@ziyik Correct! @jeongarmy Could you modify this?

Reply:
Done. thank you.